lib/Refund.php

```diff
@@ -9,6 +9,11 @@
  * been created but not yet refunded. Funds will be refunded to the credit or debit
  * card that was originally charged.
  *
+ * Stripe Tax users with recurring payments and invoices can create <a
+ * href="https://stripe.com/docs/api/credit_notes">Credit Notes</a>, which reduce
+ * overall tax liability because tax is correctly recalculated and apportioned to
+ * the related invoice.
+ *
```

Comment:
Why is this comment part of "breaking changes"?

Reply:
Responded offline.